gradle.properties

```diff
@@ -14,6 +14,8 @@ kotlin.native.ignoreIncorrectDependencies=true
 org.gradle.caching=true
 org.gradle.jvmargs=-Xmx4096m
 org.gradle.parallel=true
+# Specify a specific JAVA_HOME for gradle if needed
```

Comment:
I'm going to leave this as a comment for future reference. Let me know if there's a better alternative here